### DISCUSSION TITLE
TOTP 2FA coming to core

### DISCUSSION THREAD

**Discussion by binaryfire** [rawilk/profile-filament-plugin#65] [Q&A]
Just a heads up - TOTP 2FA is being added to core in v4. Might be worth seeing if you can hook into core's logic once it's released? PR https://github.com/filamentphp/filament/pull/14698

**Comment by rawilk** [comment]
I've refactored my MFA implementation in the plugin to behave similarly to Filament's MFA. Even though they are similar, my implementation is slightly different with the providers, so they are not compatible with each other.
